src/app/modules/header/header.component.html

```diff
@@ -1,7 +1,7 @@
 <header class="header">
   <div class="view-container header__container">
     <div class="header__logo">
-      <a class="logo" href="#">
+      <a class="logo" [routerLink]="authenticated ? ['/courses'] : ['/login']">
```

Comment:
I guess it's better to put just ['/courses'] and add guard for this page. But that's okay